pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -361,6 +364,7 @@ func (plugin *cniNetworkPlugin) buildCNIRuntimeConf(podName string, podNs string
 		ContainerID: podSandboxID.ID,
 		NetNS:       podNetnsPath,
 		IfName:      network.DefaultInterfaceName,
+		CacheDir:    plugin.cacheDir,
```

Comment:
Since [https://github.com/containernetworking/cni/pull/682](https://github.com/containernetworking/cni/pull/682) has been merged, I think we can bump up to the newest commit or wait CNI v0.7.2 release.

Reply:
@mars1024 Kube doesn't want us to depend on non-tagged releases, unfortunately. So we can't just bump to latest.

Reply:
OK, thanks for reply~